test/css_url_to_erb_asset_path_test.rb

```diff
@@ -14,6 +14,17 @@ def test_no_url
     def test_url_with_data
       css = "background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaskip);"
       assert_equal css, filter.call(css, "/current/file")
+      css = "background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaskip');"
+      assert_equal css, filter.call(css, "/current/file")
```

Comment:
I was just checking that either data( or data(' works. Because before my patch it wasn't.